bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/types/ObjectType.java

```diff
@@ -43,4 +45,11 @@
      */
     boolean isIsolated(String methodName);
 
+    /**
+     * Provides copy of type ids of the object.
+     *
+     * @return copy of type id set.
+     */
+    BTypeIdSet getTypeIdSet();
+
```

Comment:
To Complete the implementation, we need to add this to error type too.

Reply:
We already have an API in error type. Added a unit test.